apps/wpcom-block-editor/webpack.config.js

```diff
@@ -57,6 +57,10 @@ function getWebpackConfig(
 	return {
 		...webpackConfig,
 		devtool: isDevelopment ? 'inline-cheap-source-map' : false,
+		resolve: {
+			...webpackConfig.resolve,
+			modules: [ path.join( __dirname, '..', '..', 'client' ), 'node_modules' ],
```

Comment:
We might need to add root `node_modules` as well eventually? I'm not sure whether these need all need to be specified…

```suggestion
			modules: [ path.join( __dirname, '..', '..', 'client' ), 'node_modules', path.join( __dirname, '..', '..', 'node_modules' ) ],
```

Reply:
`'node_modules'` means "scan the current and all ancestor directories for `./node-modules/package-name`". It therefore needs to be specified only once.

It's documented: https://webpack.js.org/configuration/resolve/#resolvemodules